snap/hooks/prepare-device

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
A header saying that this file is included just for reference purposes and some link would be great.

Reply:
Added, let me know if that covers it/if it should be more or less/any nitpicks on wording